virtualmachine/vsphere/vm.go

```diff
@@ -1723,23 +1728,24 @@ func getVmInfo(vmMo mo.VirtualMachine, vmPath string) map[string]interface{} {
 	}
 
 	return map[string]interface{}{
-		"name":           vmPath, // full name/path of vm
-		"id":             vmMo.Self.Value,
-		"instance_uuid":  vmMo.Summary.Config.InstanceUuid,
-		"memory_size_mb": vmMo.Summary.Config.MemorySizeMB,
-		"num_cpu":        vmMo.Summary.Config.NumCpu,
-		"disks":          diskInfo,
-		"ip_addresses":   getIpFromVmMo(&vmMo),
-		"nic_info":       getNicInfo(vmMo),
-		"os_details":     getOsDetails(vmMo),
-		"tool_status":    toolsStatus,
+		"name":          vmPath, // full name/path of vm
+		"path":          vmPath, // TODO set full inventory path of vm/template
+		"id":            vmMo.Self.Value,
+		"instance_uuid": vmMo.Summary.Config.InstanceUuid,
+		"memory":        vmMo.Summary.Config.MemorySizeMB,
```

Comment:
this will affect the get_template_list call as well, make sure the controller changes are in sync. Test infra vms with this change.

Reply:
Venkat has created a ticket for controller side changes for this. Will test infra vms after changes are done in controller. Ticket: https://apporbit.atlassian.net/browse/MRPHS-4603